MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift

```diff
@@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
     
     public var dictionaryRepresentation: [String: AnyObject] {
         return [
-            "_type": "Sara6E",
+            "_type": "DailyTotal523",
```

Comment:
There are three different daily totals types for different pumps. Unfortunately, I don't have any Carelink records for that particular date.